core/modules/user/store/actions.ts

```diff
@@ -43,6 +44,17 @@ const actions: ActionTree<UserState, RootState> = {
       Vue.prototype.$bus.$emit('session-after-started')
     })
   },
+  /**
+   * Retrieve authenticated user from browser cache synchronously
+   */
+  loadFromCache (context) {
```

Comment:
we dispatch `startSession` and `loadFromCache `. Shouldn't it just work with `startSession`? Maybe we just refactor `startSession`? It's good to have this logic in single place

Reply:
Yeah, sure